packages/coding-standards/src/Finder/FileFinder.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\Finder;
+
+use IteratorAggregate;
+use SplFileInfo;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo as SymfonySplFileInfo;
+use Symplify\EasyCodingStandard\Contract\Finder\CustomSourceProviderInterface;
+
+final class FileFinder implements CustomSourceProviderInterface
+{
+    /**
+     * @param string[] $source
+     * @return IteratorAggregate
+     */
+    public function find(array $source): IteratorAggregate
+    {
+        $directories = [];
+        $files = [];
+        foreach ($source as $singleSource) {
+            if (is_file($singleSource)) {
+                $fileInfo = new SplFileInfo($singleSource);
+                $files[$fileInfo->getPath()] = new SymfonySplFileInfo($singleSource, $fileInfo->getPath(), $fileInfo->getPathname());
+            } else {
+                $directories[] = $singleSource;
+            }
+        }
+
+        $finder = Finder::create()->files()
+            ->name('#\.(twig|html\.twig|php|js)$#')
```

Comment:
I thought php files are found automatically. If so, could You remove `php`? If it works of course

Reply:
'#\.(twig|html(\.twig)?|php|js)$#'